src/components/MenuChooserTail.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+
+function MenuChooserTail(props) {
```

Comment:
Destructure at this level itself.
function MenuChooserTail ({
title,
img, 
ChoosingAppArrow
})

Reply:
addressed